### ISSUE TITLE
Error when reopening without quitting

### ISSUE THREAD

**Issue by vkozloff** [ReproNim/brainverse#193]
If I close brainverse but don't quit, then reopen the app, I get this error

<img width="417" alt="screen shot 2018-03-28 at 9 05 31 pm" src="https://user-images.githubusercontent.com/29068838/38064425-e700d884-32cb-11e8-86fd-374ec04d26f1.png">

**Comment by kaczmarj** [comment]
i get the same error on macOS 10.13.4 with brainverse 0.0.5-alpha.

**Comment by smpadhy** [comment]
Yes, you need to quit BrainVerse old version before opening it. This is due to the fact that BrainVerse listens to the port 3000. If you do not quit, port 3000 is not released to be used by the BrainVerse when you try to open.
